test/models/tweet_test.rb

```diff
@@ -25,4 +26,8 @@ class TweetTest < ActiveSupport::TestCase
     assert_not(@first_tweet.valid?)
   end
 
+  test "tweets cannot be over 124 characters" do
```

Comment:
This is probably not necessary for such a straight-forward validation, but it might be worth testing that a tweet is valid if it is under the length. Unsure if that is already covered automatically by the other tests making and saving tests though.